Foundation/CGFloat.swift

```diff
@@ -615,174 +619,178 @@ public func %=(lhs: inout CGFloat, rhs: CGFloat) {
 // tgmath
 //===----------------------------------------------------------------------===//
 
+#if os(Android)
+import Glibc
+#endif
+
 @_transparent
 public func acos(_ x: CGFloat) -> CGFloat {
-    return CGFloat(acos(x.native))
+    return CGFloat(acos(x.double))
```

Comment:
This file was pulled mostly from the overlay which means that we need to keep it in sync as the overlay changes (and for note that builds 32 bit targets so it seems we are still missing something here)